timemachine/ff/handlers/nonbonded.py

```diff
@@ -619,6 +620,7 @@ def __init__(self, patterns, params, protein_ff_name, water_ff_name, topology):
                     # Skip waters
                     continue
                 bond_idxs.append((src_atom.index, dst_atom.index))
+                bond_elements.append((src_atom.element.symbol, dst_atom.element.symbol))
```

Comment:
Should we use `element.number` instead? Would be more consistent with how we do other element checks (https://github.com/proteneer/timemachine/blob/master/timemachine/ff/handlers/nonbonded.py#L725-L731) and would probably also enable some more flexibility with using numpy arrays downstream